### DISCUSSION TITLE
CRUD operations for seller

### DISCUSSION THREAD

**Discussion by raghusaripalli** [raghusaripalli/GatorGiveaway#7] [General]
In this sprint, we shall start working on the basic functionality for the sellers. We could do create, read, update and delete for the sellers.

**Comment by tejasri555** [comment]
Yes, we can have 4 API endpoints for each of the operations and also we can upload these APIs to AWS.

**Reply by ypalnati** [reply]
All the above discussed features are implemented. We can close this thread.

**Reply by ksmSujay95** [reply]
Yeah we can close this thread as all the features discussed were implemented.
